packages/contracts-periphery/package.json

```diff
@@ -55,7 +55,7 @@
   "devDependencies": {
     "@defi-wonderland/smock": "^2.0.7",
     "@eth-optimism/contracts": "^0.5.39",
-    "@eth-optimism/contracts-bedrock": "^0.11.0",
+    "@eth-optimism/contracts-bedrock": "0.10.0",
```

Comment:
this is a hack to force yarn worskpaces to install this package in local node modules.   Otherwise it hoists both this and contract bedrock to ../../node_modules which foundry doesn't like and throws an error for if i try to remap outside of project

Reply:
you can do this as well with the `nohoist` option in the root `package.json`

Reply:
it'll create a symlink, there are some examples in the `nohoist` field